Library/Formula/docker-machine.rb

```diff
@@ -22,7 +22,9 @@ def install
     mkdir_p buildpath/"src/github.com/docker/"
     ln_sf buildpath, buildpath/"src/github.com/docker/machine"
 
+    ENV["PWD"] = buildpath/"src/github.com/docker/machine/"
```

Comment:
Change directory to here instead with a `cd "src/github.com/docker/machine/" do` block.